src/OpenTelemetry.ClrProfiler.Managed/Configuration/EnvironmentConfigurationHelper.cs

```diff
@@ -61,15 +61,15 @@ private static TracerProviderBuilder SetExporter(this TracerProviderBuilder buil
                 builder.AddConsoleExporter();
             }
 
-            switch (settings.Exporter)
+            switch (settings.TracesExporter)
```

Comment:
I think that this class is good candidate to be covered by integration tests.
Post beta/GA?

Reply:
So far the decision was to extend the exporters testing post-beta but before GA.
Related issue https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/274